Cleaning (#35)
prodmgmtprog/Product-Management-Curriculum: README.md

# Catalyst Product Management Curriculum

Developed by Founders and Coders in collaboration with CAST and Catalyst.
The Product Management Curriculum was validated as part of **Collaborative Digital Training** (May - June 2020) and the **Discovery Learning Programme** (Sept - Oct 2020).

## Learning Outcomes

- Understand the product development cycle from discovery and definition to build phase
- Understand the role and responsibilities of a Product Manager and a product owner

> - Learn methods and techniques that facilitate user-centred, consensus-driven decision making in Definition, Discovery and Build phase

Reviewer: ```suggestion
- Learn methods and techniques that facilitate a user-centred approach to Discovery, Definition, Prototyping and Building. 
```

I know you didn't write this (it was me lol) but could we say this instead?

Author: of course!

- Understand Scrum and Agile

## Goals

- Improve the skills of Product Managers
- Build the capacity of a charitable organisation \(and the charity sector\)
- Foster collaboration between charitable organisations and Product Managers

> ## How to use this book:

Author: yay!
